json-guides/bulkhead.json

```diff
@@ -12,19 +12,23 @@
             "guide_duration" : "25 minutes",
             "description": [
                 "Explore how to prevent a single failure from bringing down the entire system. MicroProfile makes this easy to do with its Fault Tolerance feature's Bulkhead policy.<br>",
-                "You'll start with a sample online banking scenario and see how it fails when no fault tolerance is implenented. You'll then enable the MicroProfile Fault Tolerance feature and use the Bulkhead policy to prevent the failing method from taking down the whole application. You will explore the two approaches to Bulkhead, semaphore isolation and thread pool isolation, and the parameters used for the annotation. Finally, you will add a fallback class which will be invoked if the BulkheadException is thrown."
+                "You'll start with a simple online banking microservice and see the shortcomings of single-threaded programs. You'll then implement concurrency to scale your microservice, and see how it fails when no fault tolerance is implemented. Next, you'll enable the MicroProfile Fault Tolerance feature and use the Bulkhead policy to prevent the failing method from taking down the whole application. You will explore the two approaches to Bulkhead, semaphore isolation and thread pool isolation, and the parameters used for the annotation. Finally, you will add a fallback class which will be invoked if the BulkheadException is thrown."
             ],
             "sections" : [
                 {
                     "name": "backgroundConcepts",
                     "title": "Background concepts",
                     "description": [
-                        "We will use the MicroProfile <b>Bulkhead</b> and <b>Asynchronous</b> policies.<br>",
+                        "We will use some concepts from Java Concurrency and the MicroProfile <b>Bulkhead</b> and <b>Asynchronous</b> policies.<br>",
```

Comment:
Instead of: We will use some concepts from Java Concurrency and the MicroProfile <b>Bulkhead</b> and <b>Asynchronous</b> policies.<br>
Maybe this: We will use some concepts from Java Concurrency along with MicroProfile <b>Bulkhead</b> and <b>Asynchronous</b> policies.<br>.